internal/pkg/manager/spod/spod_controller.go

```diff
@@ -107,6 +110,7 @@ func (r *ReconcileSPOd) Healthz(*http.Request) error {
 //
 // OpenShift (This is ignored in other distros):
 // +kubebuilder:rbac:groups=security.openshift.io,namespace="security-profiles-operator",resources=securitycontextconstraints,verbs=use
+// +kubebuilder:rbac:groups=security.openshift.io,resources=securitycontextconstraints,verbs=get;list;watch
```

Comment:
I dislike that. Do we have a better way to find out that we're running in OpenShift or not?

Reply:
In the ComplianceAsCode project, we detect OCP by looking for an instance of `apiVersion` `config.openshift.io/v1`, `Kind: ClusterOperator`, `name: openshift-apiserver`.  Looking for the instance might be an overkill, we do it to make sure that the version matches `4.y.z`, but I guess looking for `ClusterOperator` might be a bit safer than looking for `SCCs`.

Reply:
Fixed :+1: